composer/callbacks/speed_monitor.py

```diff
@@ -83,30 +86,47 @@
         'int8': 130e12,
         'int4': 260e12,
     },
+    'trn1': {
```

Comment:
Can we add a source sheet from where you got these numbers?

Reply:
Done.